.gitlab-ci.yml

```diff
@@ -1071,16 +965,21 @@ kitchen_windows-a7-2008:
   before_script:
     - rsync -azr --delete ./ $SRC_PATH
   tags: [ "runner:main", "size:large" ]
+  before_script:
+    - export WINDOWS_TESTING_S3_BUCKET=$WINDOWS_TESTING_S3_BUCKET_A7
+    - export TEST_PLATFORMS="win2008r2,MicrosoftWindowsServer:WindowsServer:2008-R2-SP1:2.127.20190603"
+  <<: *kitchen_common
+
+
+.kitchen_windows_installer_common: &kitchen_windows_installer_common
```

Comment:
nit: we could move this right after the definition of `.kitchen_common`
To have:
 - commons def
 - windows kitchen tests
 - linux kitchen tests

Reply:
done!